ci/e2e/oidc_configuration/cluster_with_managed_oidc_config/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable token {
+  type = string
+  sensitive = true
+}
+
+variable url {
+    type = string
+    default = "https://api.stage.openshift.com"
+}
+
+variable operator_role_prefix {
+    type = string
+}
+
+variable account_role_prefix {
+    type = string
+    default = ""
```

Comment:
Shouldn't we enforce the CI to give the prefix? shouldn't have default empty string as it might cause to bad behavior
The prefix is used for resources names and we need the CI to create the resources with known prefixes for better handling of undeleted resources, am I wrong?

Reply:
correct, it is required field , that should include account-role created before by the user.